app/controllers/portal.js

```diff
@@ -149,6 +76,98 @@ export default Ember.Controller.extend({
      */
     formValidation (validation) {
       this.set('isFormInvalid', validation.errors.length !== 0)
+    },
+    updateDOM: function () {
```

Comment:
This seems like The Wrong Way To Do It to me. It seems that this kind of stuff should be done declaratively with the handlebars template rather than imperatively through jQuery calls. Is there any particular reason why you chose this path?